### PR TITLE
Add hyper-highlight-active-pane plugin to plugins.json

### PR DESCRIPTION
**Pull request by yebrahim**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -577,5 +577,12 @@
     "type": "plugin",
     "preview": "https://raw.githubusercontent.com/joskore/hyper-shellect/master/shellect-preview.png",
     "dateAdded": "1546573670459"
-}
+  },
+  {
+    "name": "hyper-highlight-active-pane",
+    "description": "Highlight the currently active pane in hyperterm",
```
Comment (timothyis) [inline plugins.json]:
```suggestion
    "description": "Highlight the currently active pane in Hyper",
```

**Comment by timothyis** [comment]
You could try using [getkap.co](getkap.co) to capture it and then double the size in export (unless it automatically doubles it, if set)

**Comment by yebrahim** [comment]
Done, didn't realize I was using my external monitor, which is not retina, to record. :)

File: plugins.json [hunk 2]
```diff
@@ -577,5 +577,12 @@
     "type": "plugin",
     "preview": "https://raw.githubusercontent.com/joskore/hyper-shellect/master/shellect-preview.png",
     "dateAdded": "1546573670459"
-}
+  },
+  {
+    "name": "hyper-highlight-active-pane",
+    "description": "Highlight the currently active pane in Hyper",
+    "type": "plugin",
+    "preview": "https://raw.githubusercontent.com/yebrahim/hyper-highlight-active-pane/master/screenshot.gif",
```
Comment (timothyis) [inline plugins.json]:
```suggestion
    "preview": "https://github.com/yebrahim/hyper-highlight-active-pane/blob/39d3a1b8b264b067abd6bf1cfce2f2179f6ccf89/screenshot.gif?raw=true",
```

**Pull request merged**